### DISCUSSION TITLE
Deploy->KnowledgePlatform->CassandraDbUpdate

### DISCUSSION THREAD

**Discussion by monojit18** [Sunbird-Ed/Community#143] [Installation] [answered]
Branch: release-4.10.1_RC1
Tag: release-4.6.0_RC2

I am getting this below error consistently while running this Job.
I have seen a link at: https://github.com/project-sunbird/sunbird-community/discussions/752 - where it was suggested to block a particular trigger creation to move on.
Can you please confirm if that is still the recommended approach to make the pipeline job successful?
Please suggest.

TASK [cassandra-db-update : run cql] *******************************************
failed: [10.0.4.15] (item=data.cql) => {
    "ansible_loop_var": "item",
    "changed": true,
    "cmd": "cqlsh -f /tmp/lp-data.cql",
    "delta": "0:00:00.959492",
    "end": "2022-09-09 07:15:32.136702",
    "item": "data.cql",
    "rc": 2,
    "start": "2022-09-09 07:15:31.177210"
}

STDERR:

/tmp/lp-data.cql:47:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist
/tmp/lp-data.cql:48:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist
/tmp/lp-data.cql:49:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist
/tmp/lp-data.cql:50:ConfigurationException: Trigger class 'org.sunbird.cassandra.triggers.TransactionEventTrigger' doesn't exist

**Comment by beepdot** [comment, marked as answer]
Yes, please comment out the triggers section. It's not required.

**Reply by monojit18** [reply]
Thanks @keshavprasadms that worked!
Marking it as answered.
